src/Reader/Extension/Atom/Entry.php

```diff
@@ -610,9 +613,9 @@ protected function registerNamespaces()
     /**
      * Detect the presence of any Atom namespaces in use
      *
-     * @return string
+     * @return null|string
      */
-    protected function getAtomType()
+    protected function getAtomType(): ?string
```

Comment:
Please do not add php typehints as this will is a BC break unless the class is final.